bot/exts/code_jams/_views.py

```diff
@@ -134,3 +135,118 @@ async def interaction_check(self, interaction: discord.Interaction) -> bool:
             )
             return False
         return True
+
+
+class AddNoteModal(discord.ui.Modal, title="Add a Note for a Code Jam Participant"):
+    """A simple modal to add a note to a Jam participant."""
+
+    def __init__(self, member: discord.Member, mgmt_client: APIClient):
+        super().__init__()
+        self.member = member
+        self.mgmt_client = mgmt_client
+
+    note = discord.ui.TextInput(
+        label="Note",
+        placeholder="Your note..."
+    )
+
+    async def on_submit(self, interaction: discord.Interaction) -> None:
+        """Discord.py default to handle modal submission."""
+        try:
+            user = await self.mgmt_client.get(f"users/{quote_url(str(self.member.id))}/current_team",
+                                              raise_for_status=True)
+        except ResponseCodeError as err:
+            if err.response.status == 404:
+                await interaction.response.send_message(":x: Something went wrong, the user could not be found!")
+            else:
+                await interaction.response.send_message(":x: Something went wrong, we have notified the team!")
```

Comment:
Who does "team" refer to in this case? Maybe it's better to state it like this
```suggestion
                await interaction.response.send_message(":x: Something went wrong! Full details have been logged.")
```